### PR TITLE
Adding pip-5, address prefix

### PR DESCRIPTION
**Pull request by kehiy**

### CODE REVIEW

**Comment by Ja7ad** [comment]
@kehiy please fix pipeline errors.

**Comment by kehiy** [comment]
@Ja7ad It's done.

**Comment by Ja7ad** [comment]
@kehiy 

Please add example address of account, validator in motivation.

File: PIPs/pip-0005.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Specification
+
+to have suitable address prefix we can define `3` prefix for current types, `pv` for validators, `pc` for contracts and `pa` for account. all address must have correct prefix to be known as a valid address in `Pactus`.
+
+also the Possible future types must follow this rule and have their own prefix.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](../LICENSE).
```
Comment (Ja7ad) [inline PIPs/pip-0005.md:25]: @kehiy Don't remove `.md`

**Review by Ja7ad — CHANGES_REQUESTED** [review]

File: PIPs/pip-0005.md (hunk 1 above)
Comment (kehiy) [reply PIPs/pip-0005.md:29]: with the `.md` pipeline throws errors!

Comment (Ja7ad) [reply PIPs/pip-0005.md:29]: > with the `.md` pipeline throws errors!

If you put without `.md` link reference to branch LICENSE.

Comment (kehiy) [reply PIPs/pip-0005.md:29]: I will add `.md`, but what should we do with the pipeline?

File: PIPs/pip-0005.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: address prefix
```
Comment (Ja7ad) [inline PIPs/pip-0005.md:2]: please add:

```md
pip: 5
```

File: PIPs/pip-0005.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (Ja7ad) [inline PIPs/pip-0005.md:1]: Please change file name to:

```md
pip-0005.md to pip-5.md
```

**Review by Ja7ad — CHANGES_REQUESTED** [review]

**Comment by kehiy** [comment]
@Ja7ad done.

**Review by Ja7ad — APPROVED** [review]

**Review by parsa-hedayatnia — APPROVED** [review]

**Pull request merged**